src/components/ProfileSocialMedia/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+
+import Image from 'next/image';
+type Props = {
+  socialMedia: SocialMedia;
+};
+
+type SocialMedia = {
+  [name: string]: string;
+};
+
+const ProfileSocialMedia = ({ socialMedia }: Props) => {
+  const [invalidSocialNetworks, setInvalidSocialNetworks] = useState({});
+  const onImageError = (value: string) => {
+    setInvalidSocialNetworks({ ...invalidSocialNetworks, [value]: true });
+  };
+  return (
+    <div className="mt-2 flex items-center space-x-2">
+      {Object.keys(socialMedia).map(
+        (value) =>
+          !invalidSocialNetworks[value] && (
+            <a
+              key={value}
+              className="hover:opacity-75 bg-gray-600 rounded-full p-2 w-10 h-10"
+              href={socialMedia[value]}
+            >
+              <Image
+                src={`/img/${value}.svg`}
+                onErrorCapture={() => onImageError(value)}
+                onError={() => onImageError(value)}
+                alt={value}
+                width={24}
+                height={24}
+              />
+            </a>
+          ),
+      )}
+    </div>
+  );
+};
+
+export default ProfileSocialMedia;
```

Comment:
I saw an initial glitch on the social media icons for the profile link you shared, do you know why is it happening? Could it be related to this component?

Reply:
Ah, gotcha. Yeah, seems like a good idea, altough I think we might want to reinforce adding proper images while reviewing new profiles, which would make this case impossible to happen. But when the review is manual, then there's nothing impossible, so I like the idea of having a fallback for it.

There might be a simpler way to handle this, but for now I think it's a great start.